test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 # Requirements for tests
 wheel
 pytest
+pytest-cov
```

Comment:
Well - this is only so for the CI tests - right?  Can't we just install that in the CI?

Reply:
I've moved it to the workflow script.